space/internal/components/migration/common/common.go

```diff
@@ -0,0 +1,17 @@
+package common
+
+import (
+	"context"
+
+	"github.com/anyproto/anytype-heart/core/block/editor/smartblock"
+	"github.com/anyproto/anytype-heart/pkg/lib/database"
+)
+
+type SpaceWithCtx interface {
```

Comment:
Maybe put all common interfaces in dependencies thus we will at least have same approach inside the space module